doc/docusaurus/static/code/AuctionValidator.hs

```diff
@@ -22,23 +23,30 @@
 {-# OPTIONS_GHC -fno-strictness #-}
 {-# OPTIONS_GHC -fno-unbox-small-strict-fields #-}
 {-# OPTIONS_GHC -fno-unbox-strict-fields #-}
-{-# OPTIONS_GHC -fplugin-opt PlutusTx.Plugin:target-version=1.0.0 #-}
+{-# OPTIONS_GHC -fplugin PlutusTx.Plugin #-}
```

Comment:
This line fixes the compilation warning incorrectly saying that `plutus-tx-plugin` package is unused.

Reply:
I don't think it is worth doing this just to appease the warning: one must remember to remove this line if the plugin is no longer needed.

Reply:
Also, without this pragma the recompilation avoidance mechanism is not triggering recompilation in case of changes in the plugin/compiler.